common/shell/Shell.kt

```diff
@@ -1,13 +1,13 @@
-package com.vaticle.bazel.distribution.platform.jvm
+package com.vaticle.bazel.distribution.common.shell
```

Comment:
Moved from `platform.jvm` to `common` as it is now used by both `assemble_jvm_platform` and `deploy_npm`

The intent is for this class to be the standard for running shell commands in `bazel-distribution`.